### ISSUE TITLE
Add Google+ to agent social links

### ISSUE THREAD

**Issue by simonrimkus** [wpsight/wpcasa#14]

**Comment by chamoAt** [comment]
It would be nice to add Xing and LinkedIn too.

**Comment by jancbeck** [comment]
Can't we make these fields repeating so the user just gets to enter a URL and display text. That way they can enter whatever service they want.

**Comment by simonrimkus** [comment]
Yes, repeatable fields would definitely be the best solution. However, we need to see how to include an "infinite" list of links (and icons in some themes).

**Comment by JoeHana** [comment]
I would even rather say a "repeatable fieldgroup" - to set: icon, title, link maybe even the target (eg. _blank - as some users keep requesting)

For the theme integration I would just add a parameter to the function/shortcode which can limit the number of icons displayed.

**Comment by simonrimkus** [comment]
I'm currently reorganizing the agent profile with CMB2 fields and will see how we can improve this.

**Comment by fabianmarz** [comment]
Another advantage would be, to make the agent box output filterable. Currently, you have to override the whole template to add a new link (e.g. email address).

**Comment by florianrusch** [comment]
@JoeHana I think this can be closed. Google will have Google+ turned off soon anyway.

**Comment by fabianmarz** [comment]
@florianrusch, I think this ticket is not about adding G+ only, but adding a repeatable list of undefined services as stated in comment https://github.com/wpsight/wpcasa/issues/14#issuecomment-173879752 and the following one. So G+ is just a placeholder in this case.
